alembic/runtime/environment.py

```diff
@@ -367,7 +367,8 @@ def get_x_argument(
         The return value is a list, returned directly from the ``argparse``
         structure.  If ``as_dictionary=True`` is passed, the ``x`` arguments
         are parsed using ``key=value`` format into a dictionary that is
-        then returned.
+        then returned. If there is no `=` in the argument, value is an empty
+        string.
```

Comment:
**Michael Bayer** (zzzeek) wrote:

add "..versionchanged:: 1.13.0" with a description

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/alembic/+/4989

Reply:
**Michael Bayer** (zzzeek) wrote:

Done

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/alembic/+/4989